src/Fixer/Phpdoc/PhpdocTypesFixer.php

```diff
@@ -32,10 +33,8 @@ final class PhpdocTypesFixer extends AbstractPhpdocTypesFixer implements Configu
 {
     /**
      * Available types, grouped.
-     *
-     * @var array<string,string[]>
```

Comment:
why dropping the typing?

Reply:
It is not variable anymore.

Reply:
it's still good to have a type documented.
there is no consensus on PHPDoc how to document type of consts, but PSR proposal is here for us: https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md#433-constant-or-property

Reply:
> it's still good to have a type documented.

when it add any value, 

>  https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc-tags.md#433-constant-or-property

with the typed properties, union and intersection types and soon readonly feature I cannot imagine this is valid anymore.

But for the sake of having this approved let's have this type back.